<?xml version="1.0"?>
<launch>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find apu_medication_support)/rviz/ex_transparent.rviz" required="true" output="screen" />
  <node name="apu_identify_calendar_node" pkg="apu_medication_support" type="ex_identify_calendar" output="screen">
    <param name="first_plane_segmentation_threshold" value="0.005"/>
    <param name="outlier_removal_number_neighbors_wall" value="300"/>
    <param name="outlier_removal_th_wall" value="0.001"/>
    <param name="cluster_tolerance" value="0.006"/>
    <param name="max_cluster_size" value="60000"/>
    <param name="min_cluster_size" value="10000"/>
    <param name="outlier_removal_number_neighbors_calendar" value="1"/>
    <param name="outlier_removal_th_calendar" value="1"/>
  </node>

  <node name="apu_transparent_detection_node" pkg="apu_medication_support" type="transparent_detection" output="screen"/>
  <node name="apu_transparent_discriminator_node" pkg="apu_medication_support" type="discriminator.py" output="screen"/>
</launch>
